### PR TITLE
Loss translation errors fix

### PR DESCRIPTION
**Pull request by ndslusarz**
This has the changes fixing Issues #390 and #386 - fixing Subtraction and Addition in loss function translation so that it now works properly for Vectors and a minor change to make the python script Windows compatible.

### CODE REVIEW

**Comment by ndslusarz** [comment]
Actually I did some further testing - the small edit to make it compatible with Windows doesn't work on Windows:

```
command = " ".join([vehicle] + args)
    result = subprocess.run(command, capture_output=True, shell=True)
```

Adding `shell=True` does make it Windows friendly however the addition of `" ".join` to pass the command as string that was necessary for Linux (once the shell was set to True) once again breaks the Windows side (while the combination of `shell=True` with `command=[vehicle]+args` does work) with error:

```
Exception: Problem during compilation: The syntax of the command is incorrect.
Command was: C:\cabal\bin\vehicle.EXE compile --target LossFunction --specification vehicle-python/tests/mnist.vcl --outputFile C:\Users\MALCAD~1\AppData\Local\Temp\tmpaub_kpxhloss_function.json --network mnist:<keras.engine.sequential.Sequential object at 0x0000011537C32128>
```

Not too much of a priority I'd say, considering other issues with loss translation, since the fix is easy to do locally, but something I'll look at when I have some more time.

**Comment by MatthewDaggitt** [comment]
Okay, I've removed the command line changes and the dataset and will just merge in the loss function translation changes.
